packages/host/src/lib/csi-controller.ts

```diff
@@ -137,6 +136,7 @@ export class CSIController extends EventEmitter {
 
         this.superVisorProcess = spawn(executable, command, {
             env: {
+                ...process.env,
```

Comment:
@pietrzakacper I'm not sure if I wouldn't just remove this and pass ONLY the env we want in the runner. What are we actually using besides no-docker? I guess this will anyway be removed when we get rid of Supervisor?

Reply:
The reason it was needed is that PATH env wasn't set, so ts-node binary couldn't be found. 
I changed it to explicitly pass only PATH :wink:

Reply:
I also added DEVELOPMENT & PRODUCTION envs since it's quite useful to pass them

Reply:
@pietrzakacper yep, good idea to have DEV & PROD envs there, we use them a lot for :bug: